test/saithrift/tests/saiacl.py

```diff
@@ -1716,6 +1718,7 @@ def runTest(self):
         sai_thrift_create_neighbor(self.client, addr_family, rif_id1, ip_addr1, dmac1)
         nhop1 = sai_thrift_create_nhop(self.client, addr_family, ip_addr1, rif_id1)
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr3_subnet, ip_mask1, nhop1)
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask, rif_id1)
```

Comment:
Thanks.
'ip_addr1_subnet' is not declared. Maybe it means 'ip_addr_subnet' here?
Please also remove the local route in the cleanup stage.

Reply:
addressed comments